Sources/WalletConnectUtils/DID/DIDPKH.swift

```diff
@@ -25,8 +26,15 @@ struct DIDPKH {
         self.account = account
     }
 
-    init(account: Account) {
+    public init(account: Account) {
         self.iss = "\(DIDPKH.didPrefix):\(account.absoluteString)"
         self.account = account
     }
 }
+
+extension Account {
+
+    public var iss: String {
+        return DIDPKH(account: self).iss
+    }
```

Comment:
what is this used for? what if you want an issuer in with different did method not pkh?

Reply:
I'll rename it into `string` in another PR, to not catch the conflict